hummingbot/hummingbot
# Feat/add db migration

**Before submitting this PR, please make sure**:


**A description of the changes proposed in the pull request**:
Since we are adding a new column in the trade fees table, this PR makes the migration from the older structure of the database automatic for the user.

QA:
- Run a bot with the old version
- Update the bot to the last version and see if there is not an error in the migration
